spyder/plugins/completion/languageserver/transport/main.py

```diff
@@ -132,6 +137,23 @@ def restore(self):
     client = LanguageServerClient(zmq_in_port=args.zmq_in_port,
                                   zmq_out_port=args.zmq_out_port)
     client.start()
+
+    def watch_parent_process(pid):
+        # exist when the given pid is not alive
+        if not psutil.pid_exists(pid):
+            logger.info("parent process %s is not alive", pid)
```

Comment:
Let's change this to

    logger.info("Parent process %s is not alive, exiting!", pid)